druid/src/core.rs

```diff
@@ -715,6 +831,7 @@ impl<'a, 'b> EventCtx<'a, 'b> {
     /// Request an animation frame.
     pub fn request_anim_frame(&mut self) {
         self.base_state.request_anim = true;
+        self.base_state.needs_inval = true;
```

Comment:
This seems a bit suspicious to me. But I think I mentally have a lot of yagni about the future when we'll have finer grained invalidation. I think the thing to do is, at some point in the future, not very soon, actually implement it, then shake out all the (inevitable) places where the logic needs tightening.

Reply:
Good call: this isn't considering partial invalidation; we'll have to figure out how that should interact with animation.

The reasoning here was just that it simplified logic elsewhere, and agree we'll have to revisit when we actually implement partial inval.